src/include/daos_fs.h

```diff
@@ -217,6 +217,52 @@ dfs_mount(daos_handle_t poh, daos_handle_t coh, int flags, dfs_t **dfs);
 int
 dfs_umount(dfs_t *dfs);
 
+/**
+ * Retrieve the open pool handle on the DFS mount. This is refcounted internally and must be
+ * released with dfs_pool_put().
```

Comment:
What happens if someone unmount while some threads hold a reference with get?

Reply:
you get EBUSY